### PR TITLE
fix: escape css ident

### PR DESCRIPTION
**Pull request by inottn**
## Summary

#7719 

## Checklist

- [x] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

**Comment by ahabhgk** [comment]
Looks good to me, could you add a test case, just for avoiding regression

File: crates/rspack_plugin_css/src/parser_and_generator/mod.rs [hunk 1]
```diff
@@ -38,6 +38,9 @@ use crate::{
 static REGEX_IS_MODULES: LazyLock<Regex> =
   LazyLock::new(|| Regex::new(r"\.module(s)?\.[^.]+$").expect("Invalid regex"));
 
+static REGEX_IS_COMMENTS: LazyLock<Regex> =
+  LazyLock::new(|| Regex::new(r"/\*.*?\*/").expect("Invalid regex"));
```
Comment (JSerFeng) [inline crates/rspack_plugin_css/src/parser_and_generator/mod.rs:42]: Can this match comments with multiple lines ?

```suggestion
  LazyLock::new(|| Regex::new(r"/\*[\s\S]*?\*/").expect("Invalid regex"));
```

**Review by JSerFeng — CHANGES_REQUESTED** [review]

File: crates/rspack_plugin_css/src/parser_and_generator/mod.rs (hunk 1 above)
Comment (inottn) [reply crates/rspack_plugin_css/src/parser_and_generator/mod.rs]: Thanks, I agree.

**Review by JSerFeng — APPROVED** [review]

**Comment by JSerFeng** [comment]
Thanks !

**Pull request merged**